src/main/java/plugily/projects/buildbattle/arena/BaseArena.java

```diff
@@ -136,8 +136,7 @@ public String getPrefix() {
    */
   @NotNull
   public String getTheme() {
-    //make sure to have no NPE
-    return theme == null ? "Theme" : theme;
+    return theme;
```

Comment:
`theme` variable will never return null any more as in the setter method it is handled correctly, unless modified with reflection, _but whose going to use reflection?_